packages/ses/src/make-eval-function.js

```diff
@@ -19,7 +23,30 @@ export const makeEvalFunction = safeEvaluate => {
         // rule. Track.
         return source;
       }
-      return safeEvaluate(source);
```

Comment:
If the condition and throwing gets implemented in the safeEvaluate function, it'll cover more cases from one place I think. (as mentioned in https://github.com/endojs/endo/pull/2723/files#r1961688463)